frontend/public/components/utils/_log-window.scss

```diff
@@ -24,9 +24,16 @@ $color-log-window-divider: $color-log-window-header-bg;
   padding-top: 10px;
 }
 
-.log-window__lines {
+.log-window__lines, .log-window__lines--wrap{
```

Comment:
These changes on lines 27, 30-32 aren't necessary if you follow the suggested `classnames()` change below.  Apologies for not being more clear on that in my first round of comments.

Reply:
Made suggested changes. Resolving conversation